src/transformState.js

```diff
@@ -4,8 +4,36 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
+
+function deleteParam(state, params) {
+  delete state[params];
 }
 
+function transformState(state, actions) {
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          deleteParam(state, key);
+        }
```

Comment:
The idea was not to move the delete `object[item]` line, but to move all the logic of these actions into specific functions. To make it easier to read the code.

```
function deleteParam(state, params) {
  for (const key of params) {
    delete state[params];
  }
}
```